.editorconfig

```diff
@@ -40,6 +40,7 @@ indent_size = 2
 # Sort using and Import directives with System.* appearing first
 dotnet_sort_system_directives_first = true
 dotnet_style_require_accessibility_modifiers = always:warning
+csharp_using_directive_placement = outside_namespace:error
```

Comment:
Wrong section, should be under [*.cs]

Reply:
lol, I had it there first, but then thought it made sense to be near the using statements. I'll move it back :)